test/merkle-distributor/MerkleDistributor.ts

```diff
@@ -0,0 +1,692 @@
+import { assert } from "chai";
```

Comment:
Same question here: is this copied code or should we review it? If copied, were there any specific changes we should look at?

Reply:
The tests structure is the same, only minor things were changed: replace web3 usage with ethers and update the assertion syntax (use `expect` instead of `assert` in order to have a similar syntax with the rest of the tests)